bindings/cs/MongoDB.Libmongocrypt/CryptClient.cs

```diff
@@ -36,6 +36,13 @@ internal CryptClient(MongoCryptSafeHandle handle, Status status)
             _status = status;
         }
 
+        /// <summary>
+        /// Gets the CSFLE version.
+        /// </summary>
+        /// <returns>CSFLE version</returns>
+        public string GetCsFleVersion =>
```

Comment:
nit: I think it's better to make it property like for example `Library.Version`. Everything (99.9%) on bindings levels is about calling appropriate c method, so no need to additionally highlight it. Like `Binary.Data`, `CryptContext.State` and etc
nitx2: it looks not too long to format it in one line :)